agentos/tracker.py

```diff
@@ -31,81 +36,116 @@ class AgentTracker:
     A Component used to track Agent training and evaluation runs.
     """
 
+    DEFAULT_ENTRY_POINT = "publish"
```

Comment:
Maybe change default to `status` command.  Bigger chunk of work: moving the publish/get functionality into lower-layer PCS.